<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionsListItem</class>
 <widget class="QFrame" name="ConnectionsListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>157</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_Map">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Map</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_Offset">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_Direction">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Direction</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="button_Delete">
     <property name="toolTip">
      <string>Remove this connection</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/images.qrc">
       <normaloff>:/icons/delete.ico</normaloff>:/icons/delete.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="NoScrollComboBox" name="comboBox_Direction">
     <property name="toolTip">
      <string>Where the connected map should be positioned relative to the current map</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="NoScrollComboBox" name="comboBox_Map">
     <property name="toolTip">
      <string>The name of the map to connect to the current map</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="NoScrollSpinBox" name="spinBox_Offset">
     <property name="toolTip">
      <string>The number of spaces to move the connected map perpendicular to its connected direction</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NoScrollComboBox</class>
   <extends>QComboBox</extends>
   <header>noscrollcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>NoScrollSpinBox</class>
   <extends>QSpinBox</extends>
   <header>noscrollspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
